### PR TITLE
Add SuppressFinalPackageVersion to libraries we don't plan to ship stable yet

### PR DESCRIPTION
**Pull request by eerhardt**
- Aspire.Hosting.AWS
- OpenAI
- EventHubs

Part of #2541

FYI - @normj 
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3668)

### CODE REVIEW

File: src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <IsPackable>true</IsPackable>
+    <!-- This library needs more bake time until it can ship stable. -->
+    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```
Comment (sebastienros) [inline src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj:7]: Just for my own understanding as this is the first time I have seen this one

https://github.com/dotnet/arcade/blob/main/Documentation/CorePackages/Versioning.md?plain=1#L119-L120

**Review by sebastienros — APPROVED** [review]

**Comment by mitchdenny** [comment]
Why not Open AI?

**Review by joperezr — APPROVED** [review]
Thanks Eric!

**Comment by eerhardt** [comment]
> Why not Open AI?

The underlying client library isn't stable. https://www.nuget.org/packages/Azure.AI.OpenAI/

**Review by radical — APPROVED** [review]

**Pull request merged**